skyllh/core/datafields.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+
+"""This module provides functionality for defining which data fields of a data
+file is required at what stage.
+"""
+
+
+class DataFieldStages(
+    object,
+):
+
```

Comment:
I would add a docstring, explaining that the following constants are constructed as a binary representation

Reply:
Done.